browser/resources/tab_search/sources.gni

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+brave_tab_search_extras_grdp =
+    "$root_gen_dir/brave/browser/resources/tab_search/brave_extras.grdp"
+
+brave_tab_search_static_files = [
+  "brave/remove_me.txt",
```

Comment:
nit: remove :laughing:

Reply:
well I think it might complain if there's nothing for it to do

Reply:
And @Douglashdaniel will add a new file via https://github.com/brave/brave-core/pull/28363 (and please delete the temporary file) but I'm happy for this PR to merge directly to that branch and not hit master first as well